pt/lessons/specifics/plug.md

```diff
@@ -1,64 +1,183 @@
 ---
-version: 0.9.0
+version: 1.1.1
 title: Plug
 ---
 
-Se você estiver familiarizado com Ruby, você pode pensar sobre Plug como o Rack com uma pitada de Sinatra, ele fornece uma especificação para componentes de aplicação web e adaptadores para servidores web. Mesmo não fazendo parte do núcleo de Elixir, Plug é um projeto oficial de Elixir.
+Se você estiver familiarizado com Ruby, você pode pensar sobre Plug como o Rack com uma pitada de Sinatra. Ele fornece uma especificação para componentes de aplicação web e adaptadores para servidores web. Mesmo não fazendo parte do núcleo de Elixir, Plug é um projeto oficial de Elixir.
+
+Nós iremos começar criando uma mini aplicação web baseada no Plug. Depois, iremos aprender sobre as routas do Plug e como adicionar um Plug à uma aplicação web existente.
```

Comment:
maybe instead of `routas` use `rotas`?

Reply:
🙈 Ops! For sure. There is always some typo, it does not matter how many times we review. Thanks for pointing.

Reply:
I feel you, bro! haha Happens to me all the time haha